docs/faq.md

```diff
@@ -0,0 +1,50 @@
+### Regions = Placeholders?
+
+Yes. Regions can be understood as placeholders, i.e. sidebars, footer etc. It create a node to let the child widgets to mount onto.
```

Comment:
better avoid the word `node`, better say `It renders a placeholder component where other Widgets can mount on to`.